README.md

```diff
@@ -53,6 +53,227 @@ npm run start / npm run dev || yarn start / yarn run dev
 ```
 7. Go to [http://localhost:3000](http://localhost:3000/) in your browser.
 
+## Creating a "Reward Promotion" campaign<a id="creating-campaign"></a>
+
+### First of all, it is required to add missing ```validation_rules```:
+- Change X-App-Id to your Application ID.
+- Change X-App-Token to your Secret Key.
+- Keep the returned validation_rule id after executing each request. It will be required for the main request.
+
+```
+curl --location --request POST 'https://api.voucherify.io/v1/validation-rules' \
```

Comment:
As we discussed on Friday, a better option will be to write a small Nodejs CLI script, as then we can automate putting generated validation_rule id to campaign endpoint.

Reply:
here, you probably want to convince developer to use `missing-campaign.js` script instead of showing curls

Reply:
main README: no cURLs and
Note: The application should automatically add the missing campaign "Reward Promotion" by using script called "missing-campaign.js", so there is no need to add it manually.

README id tiered-cart-promotions:
Information previously added by Patryk + info about 'missing-campaign' + I left the cURL there "just in case", but I can also delete them from the second README.